.NET/Microsoft.Recognizers.Text.DateTime/CreatePackage.cmd

```diff
@@ -11,10 +11,10 @@ for /f "usebackq tokens=*" %%i in (`..\packages\vswhere.2.2.7\tools\vswhere -lat
 if not exist ..\nuget mkdir ..\nuget
 if exist ..\nuget\Microsoft.Recognizers.Text.DateTime*nupkg erase /s ..\nuget\Microsoft.Recognizers.Text.DateTime*nupkg
 "%MSBuildDir%\MSBuild\15.0\Bin\MSBuild.exe" /property:Configuration=release Microsoft.Recognizers.Text.DateTime.csproj
-for /f %%v in ('powershell -noprofile "(Get-Command .\bin\release\Microsoft.Recognizers.Text.dll).FileVersionInfo.FileVersion"') do set basic=%%v
-for /f %%v in ('powershell -noprofile "(Get-Command .\bin\release\Microsoft.Recognizers.Text.Number.dll).FileVersionInfo.FileVersion"') do set number=%%v
-for /f %%v in ('powershell -noprofile "(Get-Command .\bin\release\Microsoft.Recognizers.Text.NumberWithUnit.dll).FileVersionInfo.FileVersion"') do set numberWithUnit=%%v
-for /f %%v in ('powershell -noprofile "(Get-Command .\bin\release\Microsoft.Recognizers.Text.DateTime.dll).FileVersionInfo.FileVersion"') do set version=%%v
+for /f %%v in ('powershell -noprofile "(Get-Command .\bin\release\net45\Microsoft.Recognizers.Text.dll).FileVersionInfo.FileVersion"') do set basic=%%v

```

Comment:
I'd prefer if you get the version info from the net462 path. There should be no difference, but 4.6.2 is the "master" version.